vault/service/init.sls

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls syntax=yaml softtabstop=2 tabstop=2 shiftwidth=2 expandtab autoindent
+
+{% from "vault/map.jinja" import vault with context %}
+
+vault-service-init-file-managed:
+  file.managed:
+    - name: {{ vault.service.path }}
+    - source: {{ vault.service.source }}
+    - template: jinja
```

Comment:
`systemctl daemon-reload` or `initctl reload-configuration` are missing here.

Reply:
Doing that manually has not been needed for a long time with systemd:
https://github.com/saltstack/salt/blob/2014.1/salt/modules/systemd.py#L249-L250
https://github.com/saltstack/salt/blob/2014.1/salt/modules/systemd.py#L98-L105

I'll make and addition for upstart though to maintain support for Ubuntu 14.04 and older.

Reply:
Thanks. I wasn't aware of that re. systemd.